qucs/qucs/octave_window.cpp

```diff
@@ -117,7 +117,7 @@ bool OctaveWindow::startOctave()
   octProcess.waitForStarted();
 
   if(octProcess.state()!=QProcess::Running) {
-    output->setText(tr("ERROR: Failed to execute " + Program));
+    output->setText(tr("ERROR: Failed to execute ") + Program);
     return false;
```

Comment:
wouldn't be better to use [`QString::arg()`](http://doc.qt.io/qt-4.8/internationalization.html#use-qstring-arg-for-dynamic-text) so this will work well also for languages where the program name needs to be in another position (I do not have an example, but I suppose this may happen)?